request_a_govuk_domain/request/models/organisation.py

```diff
@@ -1,16 +1,6 @@
 from django.db import models
 
 
-class Organisation(models.Model):
-    name = models.CharField(unique=True)
-
-    def __str__(self):
-        return self.name
-
-    class Meta:
-        abstract = True
-
-
 class RegistrantTypeChoices(models.TextChoices):
```

Comment:
This is replicated in the front-end code. Can this definition be shared somewhere?